test/infrastructure/vcsim/controllers/controlplaneendpoint_controller.go

```diff
@@ -23,15 +23,15 @@ import (
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	kerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/klog/v2"
+	inmemoryruntime "sigs.k8s.io/cluster-api/test/infrastructure/inmemory/pkg/runtime"
+	inmemoryserver "sigs.k8s.io/cluster-api/test/infrastructure/inmemory/pkg/server"
```

Comment:
Should we move also those changes to another PR?

Reply:
valid point 👍 🏃